src/controllers/custom/getItemListsController.ts

```diff
@@ -43,7 +43,9 @@ const getItemListsController: RequestHandler = (_req, res) => {
                     item.category == "Misc" ||
                     item.category == "Resources" ||
                     item.category == "Fish" ||
-                    ((item as any).productCategory == "Pistols" && (item as MinWeapon).totalDamage == 0)
+                    ((item as any).productCategory == "Pistols" &&
```

Comment:
can we use something else than any, as any disables any type checking.

Reply:
The wonders of `warframe-items`. We will be rid of that shit at some point in the future, but for the time being, we have to accept the fact that we don't have correct typings.